sencha/translate.py

```diff
@@ -137,15 +155,19 @@ def __init__(self, args):
         if not self.peptides_are_bloom_filter:
             self.peptide_bloom_filter_filename = maybe_save_peptide_bloom_filter(
                 self.peptides,
-                self.peptide_bloom_filter,
+                peptide_bloom_filter,
                 self.alphabet,
                 self.save_peptide_bloom_filter,
             )
         else:
             self.peptide_bloom_filter_filename = self.peptides
-        self.peptide_ksize = self.peptide_bloom_filter.ksize()
+        self.peptide_ksize = peptide_bloom_filter.ksize()
         self.nucleotide_ksize = 3 * self.peptide_ksize
 
+    def set_ksizes(self, peptide_ksize):
+        self.peptide_ksize = peptide_ksize
+        self.nucleotide_ksize = 3 * peptide_ksize
```

Comment:
could this also be set in `__init__`? Or maybe we can make this into a property?

Reply:
moved